external-crates/move/crates/move-compiler/src/sui_mode/linters/mod.rs

```diff
@@ -124,17 +127,50 @@ pub fn known_filters() -> (Option<Symbol>, Vec<WarningFilter>) {
             Some(COLLECTION_EQUALITY_FILTER_NAME),
         ),
     ];
+    if INCLUDE_NEW_RULES {
+        let mut custom_filters_list = custom_filters();
+        filters.append(&mut custom_filters_list);
+    }
     (Some(ALLOW_ATTR_CATEGORY.into()), filters)
 }
 
+pub fn custom_filters() -> Vec<WarningFilter> {
+    let filters = vec![
+        WarningFilter::code(
+            Some(LINT_WARNING_PREFIX),
+            LinterDiagCategory::ConstantNaming as u8,
+            LINTER_DEFAULT_DIAG_CODE,
+            Some(CONSTANT_NAMING_FILTER_NAME),
+        ),
+        WarningFilter::code(
+            Some(LINT_WARNING_PREFIX),
+            LinterDiagCategory::ShiftOperationOverflow as u8,
+            LINTER_DEFAULT_DIAG_CODE,
+            Some(SHILF_OVERFLOW_NAME),
+        ),
+    ];
+    filters
+}
 pub fn linter_visitors() -> Vec<Visitor> {
-    vec![
+    let mut visitors = vec![
         share_owned::ShareOwnedVerifier.visitor(),
         self_transfer::SelfTransferVerifier.visitor(),
         custom_state_change::CustomStateChangeVerifier.visitor(),
         coin_field::CoinFieldVisitor.visitor(),
         freeze_wrapped::FreezeWrappedVisitor.visitor(),
         collection_equality::CollectionEqualityVisitor.visitor(),
+    ];
+    if INCLUDE_NEW_RULES {
+        let mut custom_visitor_list = custom_linter_visitors();
+        visitors.append(&mut custom_visitor_list);
+    }
```

Comment:
Please move these into the new extra lint flag that landed in #16579

Reply:
Checkout new PR for this lint #16819